openlibrary/templates/books/custom_carousel_card.html

```diff
@@ -0,0 +1,58 @@
+$def with(book, lazy, layout, key="", secondary_action=False)
+
+$ fallback_cover = 'https://openlibrary.org/images/icons/avatar_book.png'
+$ cover_host = '//covers.openlibrary.org'
+$ url = book.get('key') or book.url
+$ title = book.title
+
+$ cover_id = book.get('cover_id') or book.get('cover_i') or book.get('covers') and book['covers'][0]
+$if book.get('cover_url'):
+  $ cover_url = book.get('cover_url')
+$elif cover_id and cover_id != -1:
+  $ cover_url = '%s/b/id/%s-M.jpg'%(cover_host, cover_id)
+$elif book.get('ia'):
+  $ cover_url = '%s/b/ia/%s-M.jpg?default=%s'%(cover_host, book.get('ia')[0], fallback_cover)
+$elif book.get('cover_edition_key'):
+  $ cover_url = '%s/b/olid/%s-M.jpg'%(cover_host, book.get('cover_edition_key'))
+$else:
+  $ cover_url = False
+
+$if book.get('authors'):
+  $ author_names = [author.name or _('name missing') for author in book.authors]
+$elif book.get('author_name'):
+  $ author_names = book.get('author_name', [])
+$else:
+  $ author_names = []
+
+$if author_names:
+  $ byline = _(' by %(name)s', name=', '.join(author_names))
+$else:
+  $ byline = ''
+
+$ modifier = ''
+$ work_key = book.key if book.key.startswith('/work') else book.get('work_key')
+
+<div class="book carousel__item">
+  <div class="book-cover">
+    <a href="$(url)" data-ol-link-track="BookCarousel|CoverClick|$key">
+      $if cover_url:
+        <img class="bookcover" loading="lazy"
+          title="$('%s%s'%(title,byline))"
+          alt="$('%s%s'%(title,byline))"
+        $if lazy and layout == 'carousel':
```

Comment:
xxx

Reply:
I'm not sure if there was any action to take here.  I believe that this was marked when we were debugging the image hover issue, which was caused by an unrelated browser extension.